.github/workflows/build-test-package.yml

```diff
@@ -183,13 +184,16 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        manylinux-platform: ["_2_28-x64", "2014-x64", "_2_28-aarch64"]
-        python-version: ["37", "38", "39", "310", "311"]
+        #"_2_28-aarch64",
+        manylinux-platform: ["_2_28-x64", "2014-x64"]
+        python-version: ["38", "39", "310", "311"]
         include:
-          - python-version: 37
-            itk-package: "ITK-cp37-cp37m-manylinux"
-            c-compiler: "gcc"
-            cxx-compiler: "g++"
+          - manylinux-platform: _2_28-x64
+            target-arch: x64
+          - manylinux-platform: 2014-x64
+            target-arch: x64
+          - manylinux-platform: _2_28-aarch64
+            target-arch: aarch64
```

Comment:
May need to remove this configuration if your goal is to completely remove building for `aarch64`.